.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -0,0 +1,39 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: yaml -*-
+
+jobs:
+- job: linux
+  pool:
+    vmImage: ubuntu-16.04
+  timeoutInMinutes: 360
+  strategy:
+    maxParallel: 8
+    matrix:
```

Comment:
Is there a reason for not having Python 3.8/3.9/3.10 here? I thought they used to be built too.